docs/development/release-process.rst

```diff
@@ -64,26 +64,24 @@ Release Process
 Creating a new release
 ----------------------
 
-#. On the current pip ``master`` branch, generate a new ``AUTHORS.txt`` by
-   running ``invoke generate.authors`` and commit the results.
-#. On the current pip ``master`` branch, make a new commit which bumps the
-   version in ``pip/__init__.py`` to the release version and adjust the
-   ``CHANGES.txt`` file to reflect the current date. The release version should
-   follow a YY.N scheme, where YY is the two digit year, and N is the Nth
-   release within that year.
-#. On the current pip ``master`` branch, generate a new ``NEWS.rst`` by running
-   ``invoke generate.news`` and commit the results.
-#. Create a signed tag of the ``master`` branch of the form ``YY.N`` using the
-   command ``git tag -s YY.N``.
-#. Checkout the tag using ``git checkout YY.N`` and create the distribution
-   files using ``python setup.py sdist bdist_wheel``.
+#. Checkout the current pip ``master`` branch.
+#. Generate a new ``AUTHORS.txt`` (``invoke generate.authors``) and commit the
+   results.
+#. Bump the version in ``pip/__init__.py`` to the release version and commit
+   the results. Usually this involves dropping just the ``.devN`` suffix on the
+   version.
+#. Generate a new ``NEWS.rst`` (``invoke generate.news``) and commit the
+   results.
+#. Create a signed tag at the current commit, of the form ``YY.N``
+   (``git tag -s YY.N``).
+#. Checkout the tag (``git checkout YY.N``).
+#. Create the distribution files (``python setup.py sdist bdist_wheel``).
 #. Upload the distribution files to PyPI using twine
    (``twine upload -s dist/*``). The upload should include GPG signatures of
```

Comment:
We no longer generate GPG signatures. At least I didn't (we don't have a central key, and I don't have GPG set up on Windows). I don't know what you did for 18.0, but we should remove the reference to GPG keys.

Reply:
I didn't sign the release either. I'll remove the reference.